components/location/listItem.jsx

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable react/prop-types */
+import React, { useEffect, useState } from "react";
+
+const LocationListItem = ({
+  closePopups,
+  index,
+  location,
+  mapObject,
+  toggleLocationList,
+}) => {
+  const [onClick, setOnClick] = useState(null);
+
+  useEffect(() => {
+    if (location.marker && !onClick) {
+      // Reason why this arrow function looks so weird: https://stackoverflow.com/a/55621325
+      setOnClick(() => () => {
+        const popup = location.marker.getPopup();
+        if (!popup.isOpen()) {
+          // By including an index, we close every popup *except*
+          // for the one at the specified index.
+          closePopups(index);
+          toggleLocationList();
+          mapObject.jumpTo({
+            center: popup.getLngLat(),
+            essential: true,
+            zoom: 13,
+          });
+        } else closePopups();
+      });
+    }
+  }, [
+    closePopups,
+    index,
+    location.marker,
+    mapObject,
+    onClick,
+    toggleLocationList,
+  ]);
+
+  /** @todo
+   * - Populate card-badge-group class with content
+   */
+
+  return (
+    <div className="card location-list--item" onClick={onClick}>
+      <div className="card-content">
+        <div className="card-title">
+          {/* Kanad note: got rid of the weird `id` span and set the title to be the status label */}
```

Comment:
Yeah I think that was there cause I think there was something weird with the formatting or filtering not happening accurately without it

Reply:
Ah that makes a lot more sense! Happy to readjust this markup once we get filtering/search set up.